test/relay.test.js

```diff
@@ -0,0 +1,236 @@
+'use strict';
+
+var chai = require('chai')
+  , expect = chai.expect
+  , resolver = require('../src/resolver')
+  , helper = require('./helper')
+  , sequelize = helper.sequelize
+  , Sequelize = require('sequelize')
+  , Promise = helper.Promise
+  , attributeFields = require('../src/attributeFields');
+
+import {
+  GraphQLString,
+  GraphQLInt,
+  GraphQLFloat,
+  GraphQLNonNull,
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLList,
+  GraphQLObjectType,
+  GraphQLSchema,
+  graphql
+} from 'graphql';
+
+import {
+  sequelizeNodeInterface
+} from '../src/relay';
+
+import {
+  globalIdField,
+  connectionDefinitions,
+  connectionArgs,
+  connectionFromArray
+} from 'graphql-relay';
+
+function generateTask(id) {
+  return {
+    id: id,
+    name: Math.random().toString()
+  }
+}
+
+describe('relay', function () {
+  var User
+    , Task
+    , userType
+    , taskType
+    , taskConnection
+    , nodeInterface
+    , nodeField
+    , schema;
+
+  before(function () {
+    sequelize.modelManager.models = [];
+    sequelize.models = {};
+    User = sequelize.define('user', {
+      name: {
+        type: Sequelize.STRING
+      }
+    }, {
+      timestamps: false
+    });
+
+    Task = sequelize.define('task', {
+      name: {
+        type: Sequelize.STRING
+      }
+    }, {
+      timestamps: false
+    });
+
+    User.Tasks = User.hasMany(Task, {as: 'tasks'});
+
+    var node = sequelizeNodeInterface(sequelize, {User: userType, Task: taskType});
+    nodeInterface = node.nodeInterface;
+    nodeField = node.nodeField;
+
+    taskType = new GraphQLObjectType({
+      name: 'Task',
+      fields: {
+        id: globalIdField('Task'),
+        name: {
+          type: GraphQLString
+        }
+      },
+      interfaces: [nodeInterface]
+    });
+
+    var connection = connectionDefinitions({name: 'Task', nodeType: taskType});
+
+    taskConnection = connection.connectionType;
+
+    userType = new GraphQLObjectType({
+      name: 'User',
+      fields: {
+        id: globalIdField('User'),
+        name: {
+          type: GraphQLString
+        },
+        tasks: {
+          type: taskConnection,
+          args: connectionArgs,
+          resolve: resolver(User.Tasks)
```

Comment:
Should we bake in support for the default connection args then? before/after should be trivial i suppose, first/last is more tricky since it depends on what field should be ordered on (could be createdAt by default but then needs a syntax for changing that).

That graphql-relay test does not look to match what we have in terms of having the resolver on the top level `tasks`? Would one need to have the resolver on `edges` generally to supply extra fields?